sidekick/tests/test_views.py

```diff
@@ -63,6 +45,44 @@ def test_health_endpoint_with_vars(self):
         self.assertTrue("version" in result)
         self.assertEqual(result["version"], environ["MARATHON_APP_VERSION"])
 
+
+class SidekickAPITestCase(APITestCase):
+    def setUp(self):
+        self.api_client = APIClient()
+
+        self.user = get_user_model().objects.create_superuser(
+            username="superuser", email="superuser@email.com", password="pass"
+        )
+        token = Token.objects.create(user=self.user)
+        self.token = token.key
+        self.api_client.credentials(HTTP_AUTHORIZATION="Token " + self.token)
+
+    def mk_org(self):
+        return Organization.objects.create(
+            name="Test Organization",
+            url="http://localhost:8002/",
+            token="REPLACEME",
+            engage_url=FAKE_ENGAGE_URL,
+            engage_token="REPLACEME",
+        )
+
+
+class TestSendTemplateView(SidekickAPITestCase):
+    def add_whatsapp_messages_200_response(self, responses):
```

Comment:
I'm not sure if we have to pass responses here?

Reply:
You're right :) thanks!